src/accumulator.jl

```diff
@@ -33,7 +33,7 @@ function counter(seq)
 end
 
 eltype_for_accumulator(seq::T) where T = eltype(T)
-function eltype_for_accumulator(seq::T) where {T <: Base.Generator}
+function eltype_for_accumulator(seq:: Base.Generator}
```

Comment:
my bad, added an extra space
```suggestion
function eltype_for_accumulator(seq::Base.Generator}
```